gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/BucketRegion.java

```diff
@@ -2557,11 +2557,14 @@ protected void updateSizeOnClearRegion(int sizeBeforeClear) {
 //     }
     long oldMemValue;
 
+
+    if (!this.reservedTable() && needAccounting()) {
+      long ignoreBytes = this.isDestroyed() ? getIgnoreBytes() :
```

Comment:
I think a better check would be the one below: isDestroyed() || this.isDestroyingDiskRegion

Also what about a regular Region.close()? Should that also manipulate this?